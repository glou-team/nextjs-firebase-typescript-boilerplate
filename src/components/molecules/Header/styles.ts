import styled from 'styled-components'

export const Container = styled.header`
  width: 100%;
  height: 100px;
  border-top: 1px solid #eaeaea;
  display: flex;
  justify-content: space-around;
  align-items: center;
`
