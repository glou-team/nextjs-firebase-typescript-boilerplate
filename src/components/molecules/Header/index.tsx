import React from 'react'
import Link from 'next/link'

import * as S from './styles'

export default function Header() {
  return (
    <S.Container>
      <Link href="/">
        <a>Home</a>
      </Link>{' '}
      <Link href="/blog">
        <a>Blog</a>
      </Link>{' '}
      <Link href="/about">
        <a>About</a>
      </Link>
    </S.Container>
  )
}
