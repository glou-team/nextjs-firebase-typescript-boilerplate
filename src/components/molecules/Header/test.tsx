import { render } from '@testing-library/react'

import Header from '.'

describe('<Header />', () => {
  it('Should render the footer', () => {
    const { container } = render(<Header />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
