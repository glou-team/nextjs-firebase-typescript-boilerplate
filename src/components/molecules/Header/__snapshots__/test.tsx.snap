// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render the footer 1`] = `
.c0 {
  width: 100%;
  height: 100px;
  border-top: 1px solid #eaeaea;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<header
  class="c0"
>
  Header
</header>
`
